### PR TITLE
Add default_interface config to zeroconf

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add default_interface config to zeroconf

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35281
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/zeroconf.markdown [hunk 1]
```diff
@@ -23,3 +23,12 @@ This integration is by default enabled, unless you've disabled or removed the [`
 # Example configuration.yaml entry
 zeroconf:
 ```
+
+{% configuration %}
+zeroconf:
```
Comment (klaasnicolaas) [inline source/_integrations/zeroconf.markdown:28]:
```suggestion
zeroconf:
  description: 
  type: map
  keys:
```

File: source/_integrations/zeroconf.markdown [hunk 2]
```diff
@@ -23,3 +23,12 @@ This integration is by default enabled, unless you've disabled or removed the [`
...
+   default_interface:
+     description: By default, `zeroconf` will attempt to bind to all interfaces. For systems running using network isolation or similar, this may result in `zeroconf` being unavailable. Change this option to `true` if `zeroconf` does not function.
+     required: true
```
Comment (klaasnicolaas) [inline source/_integrations/zeroconf.markdown]:
```suggestion
     required: false
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by bdraco** [comment]
Thanks @klaasnicolaas !

**Review by frenck — APPROVED** [review]
Thanks, @bdraco! 👍

**Pull request merged**